go.mod

```diff
@@ -106,7 +106,7 @@ require (
 	k8s.io/kms v0.0.0
 	k8s.io/kube-aggregator v0.0.0
 	k8s.io/kube-controller-manager v0.0.0
-	k8s.io/kube-openapi v0.0.0-20240228011516-70dd3763d340
+	k8s.io/kube-openapi v0.0.8-gnostic.0.20240822154824-0db3e15f8b3a
```

Comment:
go ahead and rebuild this PR on the merged kube-openapi PRs and drop this tag and the replace directive below

Reply:
There are some issues with a gengo update that I'm working with Tim on resolving. Notably a deepcopy regression: https://github.com/kubernetes/kubernetes/pull/126787/commits/d960595ecd7317a26ac9f1bd008db25a2a98cc0f

Reply:
You might need to revert / rerun the bump to root out all the personal fork remnants in go.sum